### PR TITLE
chore: add volunteer JamesSimel.

### PR DESCRIPTION
**Pull request by JamesSimel**
Thanks, Team! :raised_hands:

### CODE REVIEW

File: data/users/JamesSimel.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "James Simel",
+    "bio": "Full Stack Dev",
+    "image": "https://avatars.githubusercontent.com/u/76014395?v=4",
+    "categories": [
+      {
+        "slug": "programming | software-development ",
```
Comment (codewithvoid) [inline data/users/JamesSimel.json:7]: we have categories pre-defined [here](https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json)

Updated your categories section accordingly. You can use this verbatim.
```json
  "categories": [
    {
      "slug": "programming ",
      "topics": [
        "JavaScript"
      ]
    },
    {
      "slug": "web-development",
      "topics": [
        "CSS",
        "ReactJS",
        "ExpressJS"
      ]
    },
    {
      "slug": "database-development",
      "topics": [
        "PostgreSQl"
      ]
    }
],
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @JamesSimel. Please make the suggested change. Looks good otherwise.

File: data/users/JamesSimel.json (hunk 1 above)
Comment (JamesSimel) [reply data/users/JamesSimel.json:7]: Thank you very much. Let me do that ASAP!

**Comment by JamesSimel** [comment]
Working on it

**Pull request closed without merging**